eoscostarica/guias.eoscostarica.io
# docs(frontend):Guía markdown 179

### GH Issue
Resolves #179  

### Steps to test
1. Go to Tutoriales/Markdown Guía
2.  Check content is right.
3.  Compare content with Markdown Guide

#### CheckList
- [ ] It is similar to the Markdown Guide in English
